Classes/Command/OidcCommandController.php

```diff
@@ -90,10 +90,13 @@ public function getAccessTokenCommand(string $serviceName): void
                 $serviceName,
                 $this->settings['services'][$serviceName]['options']['clientId'],
                 $this->settings['services'][$serviceName]['options']['clientSecret'],
-                'profile,name',
+                'profile name',
                 Authorization::GRANT_CLIENT_CREDENTIALS,
                 $additionalParameters
             );
+        } catch (IdentityProviderException $e) {
+            $this->outputLine('<error>%s: "%s"</error>', [$e->getMessage(), $e->getResponseBody()['error_description'] ?? '']);
```

Comment:
`$e->getResponseBody()` is an array?

Reply:
mh, yep, [apparently - or a string ](https://github.com/thephpleague/oauth2-client/blob/master/src/Provider/Exception/IdentityProviderException.php#L44)

Reply:
it is